engine/openbor.c

```diff
@@ -28504,7 +28504,7 @@ void checkdamageflip(entity* target_entity, entity *other, s_attack *attack_obje
     /* Debuging info */
     //attack_dump_object(attack_object);
 
-    target_entity->normaldamageflipdir = DIRECTION_ADJUST_OPPOSITE;
+    target_entity->normaldamageflipdir = -1;
```

Comment:
This is not an acceptable solution. We do not replace named constants with magic numbers.

Reply:
Sure

Reply:
OK, I'll look into this. For now, leave as is and I'll fix in global_beta.

Reply:
Cool thanks